src/github_review_prompts/tests/test_pr_url_parsing.py

```
@@ -0,0 +1,318 @@
+"""
+PR URL解析テスト
+grp_uvx.pyのparse_pr_url()関数とgithub_client.pyの解析機能をテスト
+"""
+
+import pytest
+import sys
+from pathlib import Path
+from typing import Optional, Tuple
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import

(F401)